### PR TITLE
Add some other components used in phpBB

### PR DESCRIPTION
**Pull request by michaelcullum**

### CODE REVIEW

File: projects/phpbb.yml [hunk 1]
```diff
@@ -15,3 +15,6 @@ components:
     - DomCrawler
     - Finder
     - Process
+    - Console
+    - HttpFoundation
+    - Request
```
Comment (xabbuh) [inline projects/phpbb.yml]: There is no "Request" component. Did you mean another one?

Comment (javiereguiluz) [inline projects/phpbb.yml]: I assume he was referring to `HttpFoundation` so we can safely remove `Request`.

Comment (michaelcullum) [inline projects/phpbb.yml]: Whoops, sorry yes.

Comment (javiereguiluz) [inline projects/phpbb.yml]: Fixed in https://github.com/symfony/symfony-marketing/commit/e27892e74268ec143e5a12435f08e443a4ff3498

**Pull request merged**